kpi/deployment_backends/kobocat_backend.py

```diff
@@ -349,3 +349,13 @@ def get_enketo_survey_links(self):
             except KeyError:
                 pass
         return links
+
+    def _submission_count(self):
+        _deployment_data = self.asset._deployment_data
+        id_string = _deployment_data['backend_response']['id_string']
+        # avoid migrations from being created for kc_reader mocked models
+        # there should be a better way to do this, right?
```

Comment:
KC uses a cache field on `XForm`: `num_of_submissions = models.IntegerField(default=0)`. Then, it has this method to get the submission count:
```
    def submission_count(self, force_update=False):
        if self.num_of_submissions == 0 or force_update:
            count = self.instances.filter(deleted_at__isnull=True).count()
            self.num_of_submissions = count
            self.save()
        return self.num_of_submissions
```
I can't find where this is actually incremented upon submission (I can see where it's updated on deletion, though), but maybe that's only because GitHub's search is bad.